### PR TITLE
ibus: 1.5.26 -> 1.5.27

### PR DESCRIPTION
**Pull request by ncfavier**
https://github.com/ibus/ibus/releases/tag/1.5.27

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ncfavier** [comment]
Seems like the `ibus-compose-locales` test is failing, but it's [already](https://hydra.nixos.org/build/188126867/log) failing on 1.5.26.

**Comment by jtojnar** [comment]
I suspect that was broken by https://github.com/NixOS/nixpkgs/pull/177318. Will probably need something like https://github.com/NixOS/nixpkgs/blob/53fcdfe1864c456d140992ee8e0ee84f8123da3f/nixos/tests/installed-tests/geocode-glib.nix#L5-L10

**Comment by ncfavier** [comment]
Those two locales aren't enough, but adding `all` does fix it.

File: pkgs/tools/inputmethods/ibus/default.nix [hunk 1]
```diff
@@ -60,22 +60,16 @@ in
...
     owner = "ibus";
     repo = "ibus";
     rev = version;
-    sha256 = "7Vuj4Gyd+dLUoCkR4SPkfGPwVQPRo2pHk0pRAsmtjxc=";
+    sha256 = "sha256-DwX7SYRb18C0Lz2ySPS3yV99Q1xQezs0Ls2P7Rbtk5Q=";
   };
 
   patches = [
-    # Fixes systemd unit installation path https://github.com/ibus/ibus/pull/2388
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/ibus/default.nix:74]: Please remove the unused fetchpatch from inputs.

Comment (ncfavier) [reply pkgs/tools/inputmethods/ibus/default.nix:74]: Done

**Pull request merged**